src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -913,28 +913,28 @@ public String toString() {
 
     //metrics below here are derived after collection
 
-    /** PF reads / total reads.  The percent of reads passing filter. */
+    /** The fraction of PF_READS/TOTAL_READS.   */
```

Comment:
in this and the following 3 metrics, please complete the sentence in english, and then provide the formula.